roottest/root/io/bigevent/CMakeLists.txt

```diff
@@ -4,22 +4,21 @@ ROOTTEST_GENERATE_DICTIONARY(libIoBigEventGeneration
   SOURCES IoBigEventGeneration.cxx
   NO_ROOTMAP
   NO_CXXMODULE
-  FIXTURES_SETUP io-bigevent-dict)
+  FIXTURES_SETUP root-io-bigevent-dict-fixture)
 
 target_link_libraries(libIoBigEventGenerationlibgen Physics)
 
 ROOTTEST_GENERATE_EXECUTABLE(IoBigEventGenerator
   IoBigEventGenerator.cxx
   LIBRARIES Core RIO Net Tree Hist MathCore libIoBigEventGenerationlibgen
-  FIXTURES_REQUIRED io-bigevent-dict
-  FIXTURES_SETUP io-bigevent-generator)
+  FIXTURES_REQUIRED root-io-bigevent-dict-fixture
+  FIXTURES_SETUP root-io-bigevent-generator-fixture)
 
 ROOTTEST_ADD_TEST(write
   COMMAND ./IoBigEventGenerator 10 1 1 1
-  FIXTURES_REQUIRED io-bigevent-generator
-  RESOURCE_LOCK io-bigevent-file)
+  FIXTURES_REQUIRED root-io-bigevent-generator-fixture
+  FIXTURES_SETUP root-io-bigevent-write-fixture)
 
 ROOTTEST_ADD_TEST(read
   COMMAND ./IoBigEventGenerator 10 1 1 20
-  FIXTURES_REQUIRED io-bigevent-generator
-  RESOURCE_LOCK io-bigevent-file)
+  FIXTURES_REQUIRED io-bigevent-generator-fixture root-io-bigevent-write-fixture)
```

Comment:
Did we mean:
```
FIXTURES_REQUIRED root-io-bigevent-generator-fixture root-io-bigevent-write-fixture)
```
?

Reply:
Yes, you are right:

https://github.com/root-project/root/pull/19052